src/MiniProfiler.Shared/ui/lib/MiniProfiler.ts

```diff
@@ -1180,6 +1180,7 @@ namespace StackExchange.Profiling {
                 this.controls = container.querySelector('mp-controls') as HTMLDivElement;
 
                 const minMax = container.querySelector('.mp-controls .mp-min-max') as HTMLSpanElement;
+                const clear = container.querySelector('.mp-controls .mp-clear') as HTMLSpanElement;
```

Comment:
This would add the listener specifically on the tiny element, rather than the whole `.mp-result` it was on before...what's the goal of the change here?

Reply:
Isn't the purpose of this `c` for clear all records? But I haven't seen any code about this C yet. Currently, events cannot be attached to non-existent elements，like `.mp-result`.
<img width="57" alt="20200419203524" src="https://user-images.githubusercontent.com/5441896/79688019-c1131e80-827d-11ea-8cff-cfb34eb8d29d.png">